src/backers.ts

```diff
@@ -14,17 +14,23 @@ export function getCollectiveBackerTiers(
   allBackers: Member[],
   backerName: string,
   backerOrganisations: string[],
-): Tier[] {
-  const tiers = allBackers.reduce<Tier[]>((acc, member) => {
-    if (!member.github) return acc
+): Tier[] | null {
+  const tiers = allBackers.reduce<Tier[] | null>((acc, member) => {
+    if (!member.github || acc === null) return acc
 
     const githubName = stripGithubName(member.github)
     const isMember = [...backerOrganisations, backerName].some(is(githubName))
 
+    /* Tiers */
     if (isMember && member.role === 'BACKER' && member.tier) {
       return uniq([...acc, member.tier])
     }
 
+    /* Admins, Contributors */
+    if (isMember && ['HOST', 'ADMIN', 'CONTRIBUTOR'].includes(member.role)) {
```

Comment:
Out of caution, I'll double check the definition of CONTRIBUTOR.

Reply:
💯

Reply:
Looks like you get the CONTRIBUTOR role when submitting an expense. So, it seems right.